charts/app/values.yaml

```diff
@@ -49,7 +49,7 @@ documentServiceApi:
       schedule: "*/1 * * * *"
       successfulJobHistoryLimit: 3
     upload:
-      maxSize: 15000000 # 15mb
+      maxSize: 15728640 # 15 mebibytes
       maxUploadAttempts: 3
```

Comment:
I'm kind of ashamed (not really) but I never knew that we call this mebibytes.

Reply:
Don't be ashamed. It's a fairly esoteric piece of knowledge. Arguably, we delivered the wrong thing by delivering what was asked for (if you know what I mean)